master/internal/config/config.go

```diff
@@ -400,33 +400,12 @@ func ReadWeight(rpName string, jobConf interface{}) float64 {
 // IsAgentRMReattachEnabled returns whether the container reattachment is enabled on AgentRM.
 func IsAgentRMReattachEnabled() bool {
```

Comment:
can you comment why this one still has to exist. (i will fix it later if you put TODO on it)

Reply:
ack